src/ui/components/Icon/styles.scss

```diff
@@ -115,3 +115,8 @@
 .Icon-featured {
   @include icon($name: "featured");
 }
+
+.Icon-restart,
```

Comment:
Oh, is it because `$name` is `restart`? Why not make the name `restart-required`?